### PR TITLE
[pcie.yaml] Move pcie configuration file path to platform directory

### PR DESCRIPTION
**Pull request by sujinmkang**
**- Why I did it**
The pcie configuration file location is under plugin directory not under platform directory.
https://github.com/Azure/sonic-buildimage/issues/6437

**- How I did it**
- Move all pcie.yaml configuration file from plugin to platform directory.
- Remove unnecessary timer to start pcie-check.service
- Move pcie-check.service to sonic-host-services

**- How to verify it**
Verify on the device.

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ x] 202012

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by jleveque — CHANGES_REQUESTED** [review]
Please make PR title more descriptive.

Also, this PR removes the systemd timer file, but there is no mention of that in the PR description. Please also enhance the PR description to detail all changes in the PR.

**Comment by lguohan** [comment]
can you have good description and pr title?

**Comment by sujinmkang** [comment]
@jleveque can you hold your review for now, I'm verifying the changes with pmon and utilities prs again.

**Comment by jleveque** [comment]
@sujinmkang: Once all of the related PRs for this change eventually merge, please be sure to update https://github.com/Azure/SONiC/wiki/Porting-Guide to reflect the path changes.

**Review by keboliu — APPROVED** [review]
LGTM

**Comment by sujinmkang** [comment]
retest vsimage please

**Comment by sujinmkang** [comment]
retest vsimage please!

**Comment by sujinmkang** [comment]
retest vsimage please

**Comment by sujinmkang** [comment]
@jleveque is the PR title still not clear?

**Comment by jleveque** [comment]
> @jleveque is the PR title still not clear?

The title and description are much better now. Thanks!

**Review by jleveque — APPROVED** [review]

File: files/image_config/pcie-check/pcie-check.sh [hunk 1]
```diff
@@ -16,11 +16,12 @@ function debug()
 
 function check_and_rescan_pcie_devices()
 {
-    PCIE_CHK_CMD='sudo pcieutil check | grep "$RESULTS"'
+    PCIE_CHK_CMD='sudo pcieutil check |grep "$RESULTS"'
```
Comment (jleveque) [inline files/image_config/pcie-check/pcie-check.sh:19]: Why remove the space?

Comment (sujinmkang) [reply files/image_config/pcie-check/pcie-check.sh:19]: I think this came from my previous commits while merging master branch. I will add it back.

**Review by jleveque — DISMISSED** [review]

**Comment by lguohan** [comment]
there is a mistake here. 

/x86_64-cel_e1031-r0pcie.yaml

**Comment by lguohan** [comment]
@sujinmkang , can you fix?

**Review by lguohan — DISMISSED** [review]
fix typo

**Comment by sujinmkang** [comment]
@lguohan  can you please review this again?

**Review by jleveque — APPROVED** [review]

**Pull request merged**